### ISSUE TITLE
Resource names have changed for 3.0.100

### ISSUE THREAD

**Issue by matkoch** [microsoft/msbuild#4852]
### Steps to reproduce

https://github.com/matkoch/msbuild-resource-names/commit/354c7fd401ec4b516b04eca3a73ad31f2340eb4d

The console application prints out all resource names:

```
// before change
➜  ClassLibrary1 git:(master) ✗ dotnet run
ClassLibrary1.templates.Build.cs
ClassLibrary1.templates.data.xml
// after change
➜  ClassLibrary1 git:(master) ✗ dotnet run
ClassLibrary1.Build
ClassLibrary1.templates.data.xml
```

Seems that the namespace is parsed to generate the resource name. From what I can tell, a breaking change for 3.0.100. Note that I also defined `<Compile Remove="templates\*.cs" />`.

@rainersigwald to be honest, this is really frustrating. I tried catching up with the latest releases, but those [embedded resource related issues](https://github.com/microsoft/msbuild/issues/4742) have been worked on at the latest possible time.

**Comment by rainersigwald** [comment]
I believe this would be fixed by #4764, which is expected to ship with SDK 3.0.101. It also shipped with Visual Studio 16.3.3.

@BenVillalobos, can you please confirm?

**Comment by BenVillalobos** [comment]
Just confirmed this, the fix will be in 3.0.101. Sorry for the trouble!

**Comment by matkoch** [comment]
Thanks
